### PR TITLE
Uploading Assignment 13 js

### PR DESCRIPTION
**Pull request by poonam-khode**

### CODE REVIEW

**Review by sanketmod — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

File: poonam/JS/Assignment13_js/que3.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <body>
+        <script>
+            var input = prompt("Enter date of birth in the following format:\nMonth, date, year ");
+            var date = new Date(input);
+            var days = prompt("Enter days: ");      
+            document.write("Date: " + date)          
+            function getDateAgo(date, days) {
+            let dateCopy = new Date(date);
+                                                                                                                                                
+            dateCopy.setDate(date.getDate() - days);
```
Comment (Snehal-113) [inline poonam/JS/Assignment13_js/que3.html]: not getting the desired output.

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: poonam/JS/Assignment13_js/que3.html [hunk 2]
```diff
@@ -7,20 +7,25 @@
...
+                dateCopy.setDate(date.getDate() - days);
+                return dateCopy.getDate();
+                }                                                                                                                                  
+                document.write("<br>Previous date: " + getDateAgo(date, days) );
+                document.write("<br>Day before previous date: " + getDateAgo(date, days+1) );
+                document.write("<br>Date: " + getDateAgo(date, 365) );
+            }
         </script>
     </body>
 </html>
```
Comment (Snehal-113) [inline poonam/JS/Assignment13_js/que3.html:31]: this program is still not working poonam, take someone's help...or check qadir or sanket's program

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: poonam/JS/Assignment13_js/que3.html (hunk 2 above)
Comment (poonam-khode) [reply poonam/JS/Assignment13_js/que3.html:31]: mam but it is working on my side

File: poonam/JS/Assignment13_js/que3.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            function output() {   
+                var input = document.getElementById("inputDate").value;
+                var days = parseInt(document.getElementById("inputDays").value); 
+                var date = new Date(input);  
+                document.write("Date: " + date)          
+                function getDateAgo(date, days) {
+                let dateCopy = new Date(date);
+                                                                                                                                                    
+                dateCopy.setDate(date.getDate() - days);
+                return dateCopy.getDate();
```
Comment (MKadeer17) [inline poonam/JS/Assignment13_js/que3.html]: Return full date instead of only date.
```suggestion
                return dateCopy;
```

**Review by MKadeer17 — APPROVED** [review]

File: poonam/JS/Assignment13_js/que3.html (hunk 3 above)
Comment (poonam-khode) [reply poonam/JS/Assignment13_js/que3.html]: thanks qadir

Comment (MKadeer17) [reply poonam/JS/Assignment13_js/que3.html]: Welcome @poonam-khode.

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**